pygtrie.py

```diff
@@ -1223,6 +1223,16 @@ def __init__(self, *args, **kwargs):
         self._separator = kwargs.pop('separator', '/')
         super(StringTrie, self).__init__(*args, **kwargs)
 
+    @property
+    def _separator(self):
+        return self.__separator
+
+    @_separator.setter
+    def _separator(self, sep):
```

Comment:
This might be tricky with all the string mess and differences between Python 2 and 3 (recall that pygtrie supports both). I don’t mind either way.